### ISSUE TITLE
Default Library?

### ISSUE THREAD

**Issue by Tharic99** [JasonHHouse/gaps#126]
Is it possible to add a feature request to specify your Default Library in the Settings under Plex? 

Then when navigating to Libraries or Missing, that Default Library will always come up first.

Thanks

**Comment by JasonHHouse** [comment]
Yup, I can add this in. Probably in the next release or two. It should be pretty straight forward.

**Comment by Tharic99** [comment]
Awesome. Looking forward to it!
